tests/Filter/ChoiceFilterTest.php

```diff
@@ -68,15 +76,15 @@ public function testFilterArrayWithNullValue(): void
         $filter->filter($builder, 'alias', 'field', ['type' => EqualOperatorType::TYPE_EQUAL, 'value' => ['1', null]]);
 
         $this->assertSame(['WHERE alias.field IN :field_name_0 OR alias.field IS NULL'], $builder->query);
-        $this->assertSame(['field_name_0' => ['1']], $builder->queryParameters);
+        $this->assertSame(['field_name_0' => ['1', null]], $builder->queryParameters);
```

Comment:
Why does `:field_name_0` include `null` now? Is this just because of code simplifications? Because I don't think it makes a difference with `null`, or?

Reply:
It's just a result of the code simplification indeed, it has no impact.